### ISSUE TITLE
PHP Variable tab key converts to HTML brackets

### ISSUE THREAD

**Issue by joelgraham** [Microsoft/vscode#3964]
When creating a new PHP variable (inside `<?php` tags), pressing tab causes the variable to be converted to HTML tags.

`$variable`

Press tab
`<1variable></1variable>`

**Comment by aeschli** [comment]
Tab behind a word invokes emmet.
@egamma Does emmet have specific php support?

**Comment by egamma** [comment]
@aeschli emmet only helps when editing html inside PHP. Enabling emmet inside PHP was a popular feature request.

**Comment by Keith-Cancel** [comment]
How can I disable this Emmet feature. It is really bugging me.

**Comment by egamma** [comment]
you can customize your keybindings by adding the following 

```json
[
    {
        "key": "tab",
        "command": "tab",
        "when": "editorTextFocus && !editorTabMovesFocus"
    },
    {
        "key": "tab",
        "command": "jumpToNextSnippetPlaceholder",
        "when": "editorTextFocus && inSnippetMode"
    },
    {
        "key": "tab",
        "command": "acceptQuickFixSuggestion",
        "when": "editorFocus && quickFixWidgetVisible"
    },
    {
        "key": "tab",
        "command": "acceptSelectedSuggestion",
        "when": "editorTextFocus && suggestWidgetVisible"
    }
]
```

**Comment by egamma** [comment]
Added a user setting `"emmet.triggerExpansionOnTab": false` that will be available in the April iteration.

**Comment by Keith-Cancel** [comment]
Sweet!

**Comment by ramya-rao-a** [comment]
The new Emmet experience is now shipped with VS Code 1.15

Tab will no longer trigger Emmet.

Please read more on what else is new at [Emmet 2.0](https://code.visualstudio.com/blogs/2017/08/07/emmet-2.0)
